pyccl/tracers.py

```diff
@@ -722,6 +757,13 @@ class NzTracer(Tracer):
     :func:`NumberCountsTracer` and :func:`WeakLensingTracer`.
     """
 
+    def __init__(self):
+        """By default this `Tracer` object will contain no actual
+        tracers
+        """
+        # Do nothing, just initialize list of tracers
+        super().__init__()
+
```

Comment:
Why do you need to add this? This seems equivalent to just letting this class use its parent's init function.